src/app_engine/gcmrecord.py

```diff
@@ -97,7 +102,9 @@ def parse_json(cls, json_string):
       content_dictionary = json.loads(json_string)
       required_keys = [
           cls.USER_ID_KEY, cls.GCM_ID_KEY, cls.CODE_KEY, cls.CODE_SENT_TIME_KEY,
-          cls.VERIFIED_KEY, cls.LAST_MODIFIED_TIME_KEY
+          cls.VERIFIED_KEY, cls.LAST_MODIFIED_TIME_KEY,
+          # TODO(tkchin): add registration id once we clear the store.
+          # cls.REGISTRATION_ID_KEY,
```

Comment:
Client will need to be updated before this change is pushed because it changes return signature for some APIs to return JSON instead of just result code. Once that happens we can push this change. Old registrations will still receive the call, just that the regid in the gcm message will be null. We should force re-registration at that point by clearing store so that the receive unintended call bug will be fixed.